lib/measures/tbd/resources/psi.rb

```diff
@@ -1374,5 +1378,7 @@ def processTBD(os_model, io_path, schema_path, psi_set)
         surface[:ratio] = ratio
       end
     end
+
+    surfaces
```

Comment:
I wasn't sure if you are building up the content of the TBD JSON in this method or not.  If so it would be good to return it here.